docker/test/integration/minifi/test/__init__.py

```diff
@@ -213,24 +245,75 @@ def __init__(self, expected_content):
         self.valid = False
         self.expected_content = expected_content
 
-    def validate(self):
+    def validate(self, dir=''):
```

Comment:
Please don't change the signature here. 

**FileOutputValidator** is the base class, where it makes more sense.